src/main/java/Event.java

```diff
@@ -0,0 +1,35 @@
+import java.time.format.DateTimeParseException;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.time.LocalDateTime;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ * Represents a Event which is a subclass of Task
+ * Overrides toString() from Task
+ */
+public class Event extends Task {
+    public LocalDate dueDate;
+    public LocalTime dueTime;
+    public Event (String name, String date) throws DateTimeParseException {
+        super(name);
+        this.dueDate = LocalDate.parse(date);
+        this.dueTime = null;
+    }
+    public Event (String name, String date, String time) throws DateTimeParseException {
+        super(name);
+        this.dueDate = LocalDate.parse(date);
+        this.dueTime = LocalTime.parse(time);
+    }
```

Comment:
Might want to include a line of space between the attributes and between methods :)